PanKanapka/bulka/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+from django.db import models
+
+class Alergeny(models.Model):
+    Ale_Id = models.IntegerField(primary_key = True)
+    Ale_Nazwa = models.TextField(null = True)
+    Ale_Ikona = models.BinaryField(null = True)
+
+class Skladniki(models.Model):
+    Skl_Id = models.IntegerField(primary_key = True)
```

Comment:
Wydaje mi się, że ID jest nadawane automatycznie, więc nie trzeba definiować tego w modelu. 
z dokumentacji django:
_By default, Django gives each model the following field:_
`id = models.AutoField(primary_key=True)`

Reply:
Zgadza się. Domyślnie są one nadawane z nazwą id, natomiast chciałam żeby nazywały się tak jak na schematach bazy danych Tomka. Usunę te definicję i będą one nadane automatycznie jak sugerujesz z nazwą id wtedy. Dodam te zmiany w kolejnym "commicie".